novawallet/Modules/Vote/Governance/TinderGov/TinderGovWireframe.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+final class TinderGovWireframe: TinderGovWireframeProtocol {
+    func back(from view: ControllerBackedProtocol?) {
```

Comment:
So, no swipe back feature? Probably, we can either talk to designers to have same back button or see what can be done in terms of overriding back button for a single screen